apis/outlook/mail/code/tool.gpt

```diff
@@ -0,0 +1,59 @@
+Name: listMailFolders
+Description: Lists all available mail folders.
+
+#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool listMailFolders
+
+---
+Name: listMessages
```

Comment:
why do you use the word message instead of email throug out this?

Reply:
An individual email is referred to as a message in the Graph API, which is why I stuck with that naming. Though I could see how it could be less helpful to the LLM. I can just update the naming within the tool.gpt file if you want, since that's all the LLM will see.